scripts/tuple_tree_generator/tuple_tree_generator/generators/typescript.py

```diff
@@ -0,0 +1,191 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+import re
+from collections import namedtuple
+from typing import Optional
+from jinja2 import Environment
+from jinja2.utils import Markup
+
+from .jinja_utils import loader
+
+from ..schema import Schema
+from ..schema import (
+    StructDefinition,
+    StructField,
+    SimpleStructField,
+    SequenceStructField,
+    ReferenceStructField,
+)
+
+int_re = re.compile(r"(u)?int(8|16|32|64)_t")
+CPP_TYPE = re.compile(r"^((?P<container>[^<>]+<(?P<contained>.*)>)|(?P<simple>[^<>]*))$")
+
+BUILTINS = ["string", "boolean", "bigint"]
+CLS_BUILTINS = ["MetaAddress"]
+ACLS_BUILTINS = ["Reference", "Identifier"]
```

Comment:
`MetaAddress` and `Identifier` cannot be hardcoded.
Use `STRING_TYPES` and `SEPARATE_STRING_TYPES` as we do for Python.

Reply:
Fixed